src/main/java/org/broadinstitute/hellbender/exceptions/UserException.java

```diff
@@ -373,6 +369,20 @@ public LexicographicallySortedSequenceDictionary(String name, SAMSequenceDiction
         }
     }
 
+    public static class SequenceDictionaryIsMissingContigLengths extends UserException {
+        private static final long serialVersionUID = 0L;
+
+        public SequenceDictionaryIsMissingContigLengths(String source, SAMSequenceDictionary dict){
+            super("GATK SequenceDictionaryValidation requires all contigs in the dictionary to have lengths associated with them.  " +
+                    "\nOne or more contigs in the dictionary from " + source + " are missing contig lengths." +
+                    "\nThe following contigs are missing lengths: " +
+                    dict.getSequences().stream()
+                            .filter( s -> s.getSequenceLength() == SAMSequenceRecord.UNKNOWN_SEQUENCE_LENGTH)
+                            .map(SAMSequenceRecord::getSequenceName)
+                            .collect(Collectors.joining(",")));
```

Comment:
Do you think we should use `.limit(20)` or something? There might be a lot...

Reply:
that's a good point